src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -210,6 +210,10 @@ private void ReAddReorganisedTransactions(Block? previousBlock)
                 for (int i = 0; i < txs.Length; i++)
                 {
                     Transaction tx = txs[i];
+                    if (tx.SupportsBlobs)
+                    {
+                        continue;
+                    }
```

Comment:
I think we want to change blob transaction handling in the tx pool PR right? @marcindsobczak ?